src/open_inwoner/conf/dev.py

```diff
@@ -75,14 +75,6 @@
 # Library settings
 #
 
-# Allow logging in with both username+password and email+password
```

Comment:
Kloppen deze dev-aanpassingen? Wil voorkomen dat je hiermee de anderen in de weg zit en lijken mij niet gerelateerd met CSP